### PR TITLE
gitlab setup

### PR DESCRIPTION
**Pull request by PavelLinearB**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=ibKx-3E)
image

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -42,16 +42,17 @@ gitStream is a workflow automation tool that enables you to use YAML configurati
...
 
 		gitStream can be installed for one repo, specific repos, or all repos in your organization. We recommend installing for **all** because it will ensure all new repos are able to use gitStream. You can change this setting at any time in the future.
 
 
 === ":fontawesome-brands-gitlab: GitLab"
 
-	Please contact LinearB 
-
-	:material-email-outline: [customers@linearb.io](mailto:customers@linearb.io)
+	1. Create an account in the <a href="https://app.linearb.io/sign-up" target="_blank">LinarB app</a>.
```
Comment (vim-zz) [inline docs/index.md:52]: 1. There is a typo, should change to `LinearB`
2. I think this text will be better "Create a free LinearB account"

File: docs/index.md [hunk 2]
```diff
@@ -42,16 +42,17 @@ gitStream is a workflow automation tool that enables you to use YAML configurati
...
+	2. Follow the steps to <a href="https://linearb.helpdocs.io/article/0xxpvue4s9-connect-git-stream-using-a-git-lab-integration" target="_blank">connect gitStream Using a GitLab Integration</a>.
```
Comment (vim-zz) [inline docs/index.md:53]: Connect gitStream to your GitLab repos by following the steps desribed in [link]

**Review by vim-zz — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (PavelLinearB) [reply docs/index.md:52]: 1. fixed
2. I thought about it, but this is not only for free accounts.

Comment (vim-zz) [reply docs/index.md]: but the account is free, its better to say free. You can say that if you already have an account you can skip that.

Comment (PavelLinearB) [reply docs/index.md]: How about: "Log in, or create a free account in the LinearB app"?

Comment (vim-zz) [reply docs/index.md]: prefect ( in → on, i.e. "on the LinearB app.")

File: docs/gitlab-installation.md [hunk 3]
```diff
@@ -7,9 +7,10 @@ description: Install gitStream to your GitLab organization.
 !!! info "Prerequisites"
 
     1. GitLab cloud
-    1. GitLab runner v15 or higher
+    2. GitLab runner v15 or higher
+    3. Create an account in the <a href="https://app.linearb.io/sign-up" target="_blank">LinarB app</a> and follow the steps to <a href="https://linearb.helpdocs.io/article/0xxpvue4s9-connect-git-stream-using-a-git-lab-integration" target="_blank">connect gitStream Using a GitLab Integration</a>.
```
Comment (vim-zz) [inline docs/gitlab-installation.md:11]: LinarB app → LinearB app

**Review by vim-zz — CHANGES_REQUESTED** [review]

**Review by vim-zz — APPROVED** [review]

**Pull request merged**